drivers/Makefile

```diff
@@ -1,12 +1,14 @@
 CC = arm-none-eabi-gcc
```

Comment:
`CC = arm-none-eabi-gcc` <- this looks hard-coded;
maybe change it to `CC ?= arm-none-eabi-gcc`, so that you can configure this externally;